WordPress/src/main/java/org/wordpress/android/ui/comments/CommentDetailFragment.java

```diff
@@ -580,7 +580,7 @@ private void showComment() {
         final TextView txtName = (TextView) getView().findViewById(R.id.text_name);
         final TextView txtDate = (TextView) getView().findViewById(R.id.text_date);
 
-        txtName.setText(mComment.hasAuthorName() ? mComment.getAuthorName() : getString(R.string.anonymous));
+        txtName.setText(mComment.hasAuthorName() ? Html.fromHtml(mComment.getAuthorName()) : getString(R.string.anonymous));
```

Comment:
Better to use `HtmlUtils.fastUnescapeHtml()` here since this string will only contains html entities (and no html markup).